### PR TITLE
Pass image into validation function.

### PR DESCRIPTION
**Pull request by sima-zhu**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.
We haven't run docker for validation in docindex main update for a while. The PR is to fix to use docker image validation. 
Here is the failed pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1689836&view=logs&j=dc056dfc-c0cf-5958-c8c4-8da4f91a3739&t=32d77e8e-e9e3-583e-42e1-947b1ce05594

Testing pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1690542&view=logs&j=dc056dfc-c0cf-5958-c8c4-8da4f91a3739&t=98c76ae6-047b-5869-f98c-bcf6099a854a
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1695349&view=results
If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by weshaggard** [comment]
@sima-zhu for future testing PR's try to open them as draft PRs.

**Comment by sima-zhu** [comment]
> @sima-zhu for future testing PR's try to open them as draft PRs.

Yes, will do

**Review by weshaggard — APPROVED** [review]
LGTM once you finish your validation

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -355,7 +355,7 @@ function PackageDependenciesResolve($artifactNamePrefix, $packageDirectory) {
 }
 
 function ValidatePackage($groupId, $artifactId, $version, $DocValidationImageId) {
-  return ValidatePackages(@{ Group = $groupId; Name = $artifactId; Version = $version; }, $DocValidationImageId)
+  return ValidatePackages @{ Group = $groupId; Name = $artifactId; Version = $version; } $DocValidationImageId
```
[the remark this answers is not in the document]
Comment (benbp) [reply eng/scripts/Language-Settings.ps1:358]: I knew I was forgetting something key in the powershell guidelines :)

**Comment by sima-zhu** [comment]
java - core - ci

**Comment by sima-zhu** [comment]
java - core - ci

**Pull request merged**